traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.HTTP.tpl.html

```diff
@@ -655,6 +655,19 @@
                     </div>
                 </div>
 
+                <div class="form-group" ng-class="{'has-error': hasError(deliveryServiceForm.consistentHashRegex), 'has-feedback': hasError(deliveryServiceForm.consistentHashRegex)}">
+                    <label class="control-label col-md-2 col-sm-2 col-xs-12">
```

Comment:
A label should have the `for` attribute that tells what it's labeling (valued as the `name` attribute of the `<input>` to which it refers)

Reply:
see my previous comment about using the `for` attribute when using `ng-model` (it references the `id` attribute of an `<input>`, which we don't use)

Reply:
`ng-model` assigns the value of a control to a property of its Javascript controller object. `for` is a pure markup attribute that should use the `name` attribute (*not* the `id`) of the control it labels. Otherwise, things that aren't a human brain with working eyes can't tell what it's labeling.

Reply:
Well, actually, it _can_ use an `id` instead of a `name`, but that's pretty much just for when the same `name` exists on multiple controls.

Reply:
Nevermind, I'm very, very wrong about this.